packages/backend/firestore.rules

```diff
@@ -1,9 +1,26 @@
 rules_version = '2';
-// Allow read/write access on all documents to any user signed in to the application
 service cloud.firestore {
   match /databases/{database}/documents {
-    match /{document=**} {
-      allow read, write: if request.auth != null;
+    // Define which users can read and write to the database
```

Comment:
We are abstracting the role of the coordinator by going automated with Cloud Functions. Consequently, at the application level, the coordinator should have no privileges above the user other than being recognized as such and being able to be enabled to call some special functionality (e.g., setup).

Following up on what was discussed via chat:
- remove ```|| request.auth.token.coordinator;```
- remove ```allow create: if request.auth != null && request.auth.token.coordinator;```
- remove `delete` in `allow create, update, delete:` for ceremonies